libs/shared-atomic-design-components/src/lib/molecules/Editable.scss

```diff
@@ -33,3 +33,17 @@
     fill: $grey-light;
   }
 }
+
+.save__button {
+  cursor: pointer;
+  margin-right: 0.2rem;
+  margin-top: -0.7rem;
+  font-size: $size-5;
+  font-weight: $weight-light;
+  color: $redi-orange-dark;
+}
+
+.save__button--disabled {
+  cursor: not-allowed;
+  color: $grey-light!important;
+}
```

Comment:
We can remove this as well since we handle `disabled` state in the `Button.scss`.